### ISSUE TITLE
Zest nodes' mouse click action done twice

### ISSUE THREAD

**Issue by thc202** [zaproxy/zaproxy#3099]
Steps to reproduce the issue:
1. Run ZAP with Zest add-on (and Script Console) installed;
2. Add the following Stand Alone Zest script:
```JSON
{
  "about": "This is a Zest script. For more details about Zest visit https://developer.mozilla.org/en-US/docs/Zest",
  "zestVersion": "0.3",
  "title": "X",
  "description": "A template standalone script",
  "prefix": "",
  "type": "StandAlone",
  "parameters": {
    "tokenStart": "{{",
    "tokenEnd": "}}",
    "tokens": {},
    "elementType": "ZestVariables"
  },
  "statements": [
    {
      "url": "http://example.com",
      "data": "",
      "method": "GET",
      "headers": "Header value",
      "assertions": [],
      "followRedirects": true,
      "cookies": [],
      "index": 1,
      "enabled": true,
      "elementType": "ZestRequest"
    }
  ],
  "authentication": [],
  "index": 0,
  "enabled": true,
  "elementType": "ZestScript"
}
```
3. Click the message's node;
4. Check the log file, there should be two similar exceptions:
```
[...]
172369 [AWT-EventQueue-0] ERROR org.zaproxy.zap.extension.zest.ZestZapUtils  - 
org.parosproxy.paros.network.HttpMalformedHeaderException
	at org.parosproxy.paros.network.HttpHeader.setMessage(Unknown Source)
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(Unknown Source)
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(Unknown Source)
	at org.parosproxy.paros.network.HttpRequestHeader.<init>(Unknown Source)
	at org.parosproxy.paros.network.HttpMessage.setRequestHeader(Unknown Source)
	at org.zaproxy.zap.extension.zest.ZestZapUtils.toHttpMessage(Unknown Source)
	at org.zaproxy.zap.extension.zest.ExtensionZest.displayMessage(Unknown Source)
	at org.zaproxy.zap.extension.zest.dialogs.ZestDialogManager$1.mouseClicked(Unknown Source)
	at org.zaproxy.zap.extension.zest.dialogs.ZestDialogManager$1.mouseReleased(Unknown Source)
[...]
172387 [AWT-EventQueue-0] ERROR org.zaproxy.zap.extension.zest.ZestZapUtils  - 
org.parosproxy.paros.network.HttpMalformedHeaderException
	at org.parosproxy.paros.network.HttpHeader.setMessage(Unknown Source)
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(Unknown Source)
	at org.parosproxy.paros.network.HttpRequestHeader.setMessage(Unknown Source)
	at org.parosproxy.paros.network.HttpRequestHeader.<init>(Unknown Source)
	at org.parosproxy.paros.network.HttpMessage.setRequestHeader(Unknown Source)
	at org.zaproxy.zap.extension.zest.ZestZapUtils.toHttpMessage(Unknown Source)
	at org.zaproxy.zap.extension.zest.ExtensionZest.displayMessage(Unknown Source)
	at org.zaproxy.zap.extension.zest.dialogs.ZestDialogManager$1.mouseClicked(Unknown Source)
[...]
```

ZAP:
Version 2.5.0.

Add-On:
Zest version 22.

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#684.

**Comment by thc202** [comment]
Released in version 23 of Zest add-on (bundled in ZAP 2.6.0).
